src/App.js

```diff
@@ -1344,10 +1415,20 @@ render() {
 
                   <Balance icon={xdai} selected={selected} text={"xDai"} amount={this.state.xdaiBalance} address={account} dollarDisplay={dollarDisplay}/>
                   <Ruler/>
+
+
+
+
                   <Balance icon={dai} selected={selected} text={"DAI"} amount={this.state.daiBalance} address={account} dollarDisplay={dollarDisplay}/>
                   <Ruler/>
                   <Balance icon={eth} selected={selected} text={"ETH"} amount={parseFloat(this.state.ethBalance) * parseFloat(this.state.ethprice)} address={account} dollarDisplay={dollarDisplay}/>
                   <Ruler/>
+                  <div style={{cursor:"pointer"}} onClick={()=>{
```

Comment:
Badge.js and Badges.js are already nicely integrated into the UI of the burner wallet.

For a client, I used these components to display NFTs below the balances on the main screen (cannes.motion.ooo):
<img width="833" alt="Screen Shot 2019-06-05 at 13 35 57" src="https://user-images.githubusercontent.com/2758453/58953487-f06c8a00-8796-11e9-9f28-b04cc084cbb5.png">

I wonder if we could do the same in this PR instead of redirecting the user to another page.

Reply:
The issue specifically gave me a component to use.